linera-views/src/key_value_store_view.rs

```diff
@@ -560,6 +558,40 @@ where
         Ok(self.context.read_value_bytes(&key).await?)
     }
 
+    /// Obtains the value at the given index, if any.
+    /// ```rust
+    /// # tokio_test::block_on(async {
+    /// # use linera_views::memory::create_memory_context;
+    /// # use linera_views::key_value_store_view::KeyValueStoreView;
+    /// # use crate::linera_views::views::View;
+    /// # let context = create_memory_context();
+    ///   let mut view = KeyValueStoreView::load(context).await.unwrap();
+    ///   view.insert(vec![0,1], vec![42]).await.unwrap();
+    ///   assert!(view.test_existence(&[0,1]).await.unwrap());
+    ///   assert!(!view.test_existence(&[0,2]).await.unwrap());
+    /// # })
+    /// ```
+    pub async fn test_existence(&self, index: &[u8]) -> Result<bool, ViewError> {
```

Comment:
Is there a reason for the different name?

Reply:
No, and we use `contains_key` now.